README.md

```diff
@@ -84,7 +84,7 @@ self.client = [PTPusher pusehrWithKey:@"YOUR_API_KEY" delegate:self encrypt:YES
 
 ### Subscribe to channels
 
-Channels are a way of filtering the events you want your application to receive. In addition, private and presence channels can be used to control access to events and in the case of presence channels, see who else is subscribing to events. For more information on channels, see the Pusher documentation.
+Channels are a way of filtering the events you want your application to receive. In addition, private and presence channels can be used to control access to events and in the case of presence channels, see who else is subscribing to events. For more information on channels, see the [Pusher Channels documentation](https://pusher.com/docs/channels).
```

Comment:
Can I suggest adding a leading sentence here that explains that Pusher Channels uses a concept called Channels - or similar, just to establish the difference? :cry:

Reply:
I think it's OK as it is. I don't think these docs should go into too much detail about explaining the Channels product overall. That's what the docs at `pusher.com/docs/channels` are for. These docs should just explain how to use this library.